system-contracts/contracts/interfaces/ICompressor.sol

```diff
@@ -19,12 +19,12 @@ interface ICompressor {
     function publishCompressedBytecode(
         bytes calldata _bytecode,
         bytes calldata _rawCompressedData
-    ) external payable returns (bytes32 bytecodeHash);
```

Comment:
Please consider the comments made in the review of `system-contracts/contracts/Compressor.sol`